rpcserver.go

```diff
@@ -39,6 +42,28 @@ import (
 
 var (
 	defaultAccount uint32 = waddrmgr.DefaultAccountNum
+
+	// roPermissions is a slice of method names that are considered "read-only"
+	// for authorization purposes, all lowercase.
+	roPermissions = []string{
+		"verifymessage",
+		"getinfo",
+		"listpeers",
+		"walletbalance",
+		"channelbalance",
+		"pendingchannels",
+		"listchannels",
+		"lookupinvoice",
```

Comment:
Would like some feedback here: should I merge the `lookupinvoice` and `listinvoices` permissions? What about the `listchannels` and `pendingchannels` permissions? Any others that I'm missing that might be good to merge?